packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec

```diff
@@ -8,7 +8,7 @@
 %global foreman_min_version 2.4
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.1
+Version: 1.0.2
 Release: 1%{?foremandist}%{?dist}
 Summary: Adds puppet ENC features
```

Comment:
I think this isn't entirely accurate anymore since it does more.

Reply:
Happy to change that in the gem, but now it is the gem's summary :shrug: